paper/src/main/java/com/programmerdan/minecraft/simpleadminhacks/configs/OneTimeTeleportConfig.java

```diff
@@ -47,6 +50,15 @@ protected void wireup(ConfigurationSection config) {
 			}
 			materialBlacklist.add(material);
 		}
+		this.worldBlacklist = new ArrayList<>();
+		List<String> worlds = config.getStringList("home_world_blacklist");
+		for (String world : worlds) {
+			World realWorld = Bukkit.getWorld(world);
+			if (realWorld == null) {
+				continue;
+			}
```

Comment:
I don't think there's any need to check whether the world exists on wireup. If it doesn't exist then it'll never match with a given location's world. You'd only really want this if you wanted to log a warning about an unknown world name in the config.